index.bs

```diff
@@ -5820,30 +5820,51 @@ Note: In order to interoperate, user agents storing large blobs on authenticator
 :: `largeBlob`
 
 : Operation applicability
-:: [=authentication extension|Authentication=]
+:: [=registration extension|Registration=] and [=authentication extension|authentication=]
 
 : Client extension input
 ::  <xmp class="idl">
     partial dictionary AuthenticationExtensionsClientInputs {
         AuthenticationExtensionsLargeBlobInputs largeBlob;
     };
 
+    enum LargeBlobSupport {
+      "required",
+      "preferred",
+    };
+
     dictionary AuthenticationExtensionsLargeBlobInputs {
+        DOMString support;
         boolean read;
         ArrayBuffer write;
     };
     </xmp>
 
     <div dfn-type="dict-member" dfn-for="AuthenticationExtensionsLargeBlobInputs">
+        :   <dfn>support</dfn>
+        ::  A DOMString that takes one of the values of {{LargeBlobSupport}}. (See [[#sct-domstring-backwards-compatibility]].) Only valid during [=registration extension|registration=].
+
         :   <dfn>read</dfn>
-        ::  A boolean that indicates that the [=[RP]=] would like to fetch the previously-written blob associated with the asserted credential.
+        ::  A boolean that indicates that the [=[RP]=] would like to fetch the previously-written blob associated with the asserted credential. Only valid during [=authentication extension|authentication=].
 
         :   <dfn>write</dfn>
-        ::  An opaque byte string that the [=[RP]=] wishes to store with the existing credential.
+        ::  An opaque byte string that the [=[RP]=] wishes to store with the existing credential. Only valid during [=authentication extension|authentication=].
     </div>
 
+: Client extension processing ([=registration extension|registration=])
```

Comment:
We should indicate that creating a credential with the `largeBlobs` extension is required in order to store and read a blob on assertion later regardless of authenticator support. Otherwise, we still have to be Cute (tm) and always attempt to create a largeBlobKey.

Reply:
Done.